kitsune/questions/views.py

```diff
@@ -514,32 +506,20 @@ def aaq(
 
         return HttpResponseRedirect(path)
 
-    if product_key is None:
+    user_agent = request.META.get("HTTP_USER_AGENT", "")
+    is_mobile_device = parse_ua(user_agent).is_mobile
```

Comment:
Let's use `get_user_agent` instead which checks cache first.

Reply:
Oh fair enough, I couldn't work out why you were using that module when we weren't using the middleware - it looks like we can use the caching without the middleware though.